tests/pages/components/editor-with-preview.js

```diff
@@ -28,20 +26,12 @@ export default {
     isActive: hasClass('active')
   },
 
-  focus: triggerable('focus', 'textarea'),
+  textarea: submittableTextarea,
```

Comment:
Extracted the textarea into a page object of its own, since it already is a component of its own